Database/Models/Economy/Tax.cs

```diff
@@ -0,0 +1,35 @@
+using System.Text.Json.Serialization;
+using System.ComponentModel.DataAnnotations.Schema;
+using SpookVooper_2.Database.Models.Entities;
+
+namespace SpookVooper_2.Database.Models.Economy;
+
+public enum TaxType
+{
+    // PersonalIncome and CorporateIncome are paid daily
+    Transactional,
+    Sales,
+    PersonalIncome,
+    CorporateIncome,
+    CompanyIncome,
+    Payroll,
+    None
+}
+
+public class Tax : Entity
+{
+    public decimal Rate { get; set;}
+    public bool DistrictTax { get; set;}
```

Comment:
DistrictTaxPolicy should probably extent TaxPolicy

Reply:
why would districtaxpolicy be different from TaxPolicy? seems kinda dumb to create another db table.